webapp/packages/plugin-sql-editor/src/ISqlEditorTabState.ts

```diff
@@ -13,6 +13,8 @@ export interface IResultTab {
   // new group id generates every time you execute query in new tab
   groupId: string;
   indexInResultSet: number;
+  presentationId: string;
+  valuePresentationId: string | null;
```

Comment:
please take a look at:
[private async handleTabRestore(tab: ITab<ISqlEditorTabState>): Promise<boolean> {](https://github.com/dbeaver/cloudbeaver/blob/38987293a31323662fcc3efb96cdb1cb942835fc/webapp/packages/plugin-sql-editor-navigation-tab/src/SqlEditorTabService.ts#L275)

probably we want to use `zod` here

Reply:
done